packages/@sanity/cli/src/actions/init-project/templates/nextjs/index.ts

```diff
@@ -165,6 +165,12 @@ export const client = createClient({
 })
 `
 
+const live = `import { defineLive } from "next-sanity";
```

Comment:
Maybe we should add a description to the top for what this file is for?
```suggestion
// Importing and querying through `sanityFetch` from this file will make data automatically stay up to date with any changes.
// Before using it, import and render `<SanityLive />` in your layout - see
// https://github.com/sanity-io/next-sanity#live-content-api for more information.
const live = `import { defineLive } from "next-sanity";
```